modules/apps/layout/layout-seo-service/src/main/java/com/liferay/layout/seo/internal/configuration/LayoutSEOCompanyConfiguration.java

```diff
@@ -57,4 +57,15 @@
 	)
 	public boolean enableOpenGraph();
 
+	/**
+	 * Sets if open graph is enabled.
+	 */
+	@Meta.AD(
+		deflt = "false",
+		description = "layout-seo-configuration-enable-only-translated-hreflang-description",
+		name = "layout-seo-configuration-enable-only-translated-hreflang",
+		required = false
+	)
+	public boolean enableOnlyTranslatedHreflang();
```

Comment:
This sounds very weird. Maybe something like `useLanguagesWithTranslations` or something like that?

Reply:
Finally >" LayoutTranslatedLanguages"